test/suite/spring/jx_create_spring.go

```diff
@@ -13,20 +13,18 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var (
-	SkipManualPromotion = os.Getenv("JX_BDD_SKIP_MANUAL_PROMOTION")
-)
+var SkipManualPromotion = os.Getenv("JX_BDD_SKIP_MANUAL_PROMOTION")
 
 var _ = Describe("create spring\n", func() {
 	var T SpringTestOptions
 
 	BeforeEach(func() {
 		T = SpringTestOptions{
-
 			helpers.TestOptions{
 				ApplicationName: helpers.TempDirPrefix + "spring-" + strconv.FormatInt(GinkgoRandomSeed(), 10),
 				WorkDir:         helpers.WorkDir,
 				JavaVersion:     "11",
+				ProjectType:     "maven-project",
```

Comment:
Would be good to parametrize this, instead of hardcoding it. Will do it next week.